creator-node/src/config.js

```diff
@@ -333,6 +333,12 @@ const config = convict({
     format: String,
     env: 'trackBlacklist',
     default: ''
+  },
+  rehydrateConcurrencyCount: {
```

Comment:
Can we name this following the same pattern as the others?

`transcodingMaxConcurrency`